app/helpers/organisation_helper.rb

```diff
@@ -49,4 +52,9 @@ def school_capacity(school)
   def pupils
     t('schools.size.pupils').pluralize
   end
+
+  def school_type_with_religious_character(school)
+    school_type = school.school_type.label.singularize
+    school.has_religious_character? ? school_type + ', ' + school.gias_data['ReligiousCharacter (name)'] : school_type
```

Comment:
i think the convention is to use string templating (like line 17 of next file) rather than concatenation but @cesidio or @cpjmcquillan are better placed to confirm this

Reply:
I believe this method already existed, it's just been moved into the organisation helper and made into a private method.